modules/installation-configuration-parameters.adoc

```diff
@@ -2342,7 +2342,7 @@ The CRN must be enclosed in quotes ("").
     ibmcloud:
       bootVolume:
         encryptionKey:
-|A Key Protect or Hyper Protect Crypto Services root key that should be used to encrypt the root (boot) volume of only compute machines.
+|A Key Protect root key that should be used to encrypt the root (boot) volume of only compute machines.
```

Comment:
"Should be used" is not really good documentation style. It is not clear exactly who or what "should" use the key and also why it is "should be used" and not just "is used".

I can't make a suggestion because I can't see the context. A preview link is not provided and I was unable to find the assembly in the full preview.

Reply:
Passive tense is perfectly acceptable here.